README.md

```diff
@@ -55,19 +55,24 @@ It's possible to run Panoptes only having to install the `fig_rake` gem. Alterna
 0. `cd` into the cloned folder. Run either `bundle install` or `gem install fig_rake`
 
 0. Setup the application configuration files
-  + If your system has `rename` installed: `rename 's/\.yml.hudson$/\.yml/' config/*.yml.hudson`
```

Comment:
Agreed on that, that's why I installed rename instead of copy/pasting magic. Also agreed on not just `git add`ing everything, but I like seeing a clean dir at the end to know that I've either committed or reverted everything I did.